KioskApplication/src/main/java/controller/MainWindowController.java

```diff
@@ -95,10 +100,16 @@ public void switchToScreen(ApplicationScreen screen) {
             case ADMIN_MENU:
                 switchButton.setText("Logoff");
                 switchButton.requestFocus();
+                //shows all but the Map tab for logged in Users
+                tabPane.getTabs().remove(tabMap);
+                tabPane.getTabs().addAll(tabMB,tabRM,tabRS,tabSettings);
                 break;
             case PATHFINDING:
-                switchButton.setText("Employee login");
+                switchButton.setText("Staff Login");
                 switchButton.requestFocus();
+                //hides all but the Map tab from non logged in users
+                tabPane.getTabs().removeAll(tabMap,tabMB,tabRM,tabRS,tabSettings);
+                tabPane.getTabs().add(tabMap);
```

Comment:
This is great! We need to find a way for the admin to also have the map screen but otherwise this works perfectly.

Reply:
This is the last change I'll make to this branch, fair warning though, this may change a lot about the MainWindowController because the current implementation is hackey to say the least